adapters/LiftoffMonetize/LiftoffMonetizeAdapter/GADMAdapterVungleUtils.m

```diff
@@ -25,48 +25,12 @@
   return error;
 }
 
-const CGSize kVNGBannerShortSize = {300, 50};
-GADAdSize GADMAdapterVungleAdSizeForAdSize(GADAdSize adSize) {
-  if (adSize.size.height >= GADAdSizeMediumRectangle.size.height &&
-      adSize.size.width >= GADAdSizeMediumRectangle.size.width) {
-    return GADAdSizeMediumRectangle;
+VungleAdSize *_Nonnull GADMAdapterVungleConvertGADAdSizeToBannerSize(GADAdSize adSize) {
+  if (adSize.size.height == 0) {
+    return [VungleAdSize VungleAdSizeWithWidth:adSize.size.width];
   }
 
-  // An array of supported ad sizes.
-  GADAdSize shortBannerSize = GADAdSizeFromCGSize(kVNGBannerShortSize);
-  NSArray<NSValue *> *potentials = @[
-    NSValueFromGADAdSize(GADAdSizeBanner), NSValueFromGADAdSize(GADAdSizeLeaderboard),
-    NSValueFromGADAdSize(shortBannerSize)
-  ];
-
-  GADAdSize closestSize = GADClosestValidSizeForAdSizes(adSize, potentials);
-  CGSize size = CGSizeFromGADAdSize(closestSize);
-  if (size.height == GADAdSizeBanner.size.height) {
-    if (size.width < GADAdSizeBanner.size.width) {
-      return shortBannerSize;
-    } else {
-      return GADAdSizeBanner;
-    }
-  } else if (size.height == GADAdSizeLeaderboard.size.height) {
-    return GADAdSizeLeaderboard;
-  }
-  return GADAdSizeInvalid;
-}
-
-BannerSize GADMAdapterVungleConvertGADAdSizeToBannerSize(GADAdSize adSize) {
-  if (GADAdSizeEqualToSize(adSize, GADAdSizeMediumRectangle)) {
-    return BannerSizeMrec;
-  }
-  if (adSize.size.height == GADAdSizeLeaderboard.size.height) {
-    return BannerSizeLeaderboard;
-  }
-
-  // Vungle SDK will try to fit the banner in the view, but the true height of the asset
-  // is 50px since this is the only supported size.
-  if (adSize.size.width < GADAdSizeBanner.size.width) {
-    return BannerSizeShort;
-  }
-  return BannerSizeRegular;
+  return [VungleAdSize VungleAdSizeFromCGSize:adSize.size];
```

Comment:
Just want to clarify here,
We don't support standard banner size anymore? Or, standard banner sizes will be supported by "custom size" ?

Reply:
Yes it will be supported via Custom size api. Since platform no longer need the adsize string to see which one is which, SDK no longer need to depend on those apis. Pubs can now use Custom size api for all the fixed sizes.
@ashinagawa

Reply:
Just want to make sure.
So, if a pub can use normal banner (not in_feed) Placement for this class? Won't requesting standard banner with `VungleAdSizeFromCGSize` through `error` when requested?

Reply:
Discussed with Manoj and this looks good.